MapboxCoreNavigation/ProcessInfo.swift

```diff
@@ -0,0 +1,24 @@
+#if os(iOS) || os(tvOS)
+    import UIKit
+#elseif os(watchOS)
+    import WatchKit
+#endif
+
+extension ProcessInfo {
+    static var systemName: String {
+        #if os(iOS) || os(tvOS)
+            return UIDevice.current.systemName
+        #elseif os(watchOS)
+            return WKInterfaceDevice.current.systemVersion
```

Comment:
The iOS/tvOS code uses `UIDevice.systemName`, which returns either “iOS” or “tvOS”. But this line uses `WKInterfaceDevice.systemVersion`, which returns a version number like 2.0. Did you mean `systemName`, which returns “watchOS”, instead?

Maybe this method would be more understandable if we hard-code the values we want it to return.